tests/e2e/lib.sh

```diff
@@ -169,9 +169,9 @@ deploy_stackrox_operator() {
         # Retrieving values from json map for operator and iib
         ocp_version=$(kubectl get clusterversion -o=jsonpath='{.items[0].status.desired.version}' | cut -d '.' -f 1,2)
         OPERATOR_VERSION=$(< operator/midstream/iib.json jq -r '.operator.version')
-        VERSION=$(< operator/midstream/iib.json jq -r --arg version "$ocp_version" '.iibs[$version]')
+        VERSION=v${ocp_version}
```

Comment:
No, i will remove it in the PR

Reply:
I would suggest to keep the `operator/midstream/iib.json` even if not used. Because this file will be helpful if we want to debug things with any specific build

Reply:
We can keep the iib file for now, and look too removed it in a future PR if needed. @robbycochran @pratham-m